tests/test_configuration.py

```diff
@@ -204,12 +204,32 @@ def test_propagation_baggage(self):
             self.assertIsNone(carrier.get("x-b3-traceid"))
             self.assertEqual(carrier.get("baggage"), "abc=def")
 
+    def test_propagation_baggage(self):
```

Comment:
There is already a `test_propagation_baggage` function.

```suggestion
    def test_propagation_tracecontext(self):
```